### DISCUSSION TITLE
Failed to sync (OneDrive)

### DISCUSSION THREAD

**Discussion by AggroMonkey** [remotely-save/remotely-save#801] [Q&A]
I'm new to Obsidian and plug-ins in general. I've recently gotten Remotely Save to sync with my vault on my phone using OneDrive. on my PC I'm able to see the folders and files I have on my phone, but after that, it has failed to sync between my 2 devices. My phone states I have a successful sync but my PC says it has failed to sync.

**Comment by fyears** [comment]
> my PC says it has failed to sync.

are you using official onedrive app or obsidian with remotely savd on PC?

what’s the error message?
